stock-quote/src/main/java/com/whz/stockquote/Main.java

```diff
@@ -1,24 +1,84 @@
+/*
+ * Copyright © 2020-2021, Fachgruppe Informatik WHZ <help.flaxel@gmail.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 package com.whz.stockquote;
 
-import java.io.IOException;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
+/**
+ * This class represents the main class for the stock quote application. It can be used to start the
+ * program.
+ *
+ * @since 1.0.0
+ */
 public class Main {
 
-  public static void main(String[] args) throws IOException, InterruptedException {
-    String serviceName = System.getenv("RABBITMQ_SERVICE");
-    String exchangeName = System.getenv("RABBITMQ_EXCHANGE");
-    String exchangeType = System.getenv("RABBITMQ_EXCHANGE_TYPE");
+  private static String serviceName;
+  private static String exchangeName;
+  private static boolean durableExchange = false;
+  private static int publishInterval;
+  private static boolean publishRandomly = false;
```

Comment:
Do the variables have to be initialised here when they are overwritten in the methods?